packages/roosterjs-editor-core/lib/editor/Editor.ts

```diff
@@ -374,7 +377,7 @@ export default class Editor {
         return content;
     }
 
-    private getColorNormalizedContent(content: string): string {
+    public getColorNormalizedContent(content: string): string {

```

Comment:
move this function and isDataAttributeSettable() out to a util function since they don't rely on editor